dvc/system.py

```diff
@@ -146,10 +146,9 @@ def reflink(source, link_name):
 
     @staticmethod
     def _getdirinfo(path):
```

Comment:
look like name should include win in it since the function is win specific?

Reply:
@shcheklein there is no getdirinfo for non-windows, so not really, since it is apparent that is used in inode() for windows.

Reply:
It's a matter or readability ... I can say that I spent a few cycles since name didn't match for me ... I was expecting to see ifdef or something .. up to you though

Reply:
system.py can take quite a bit of refactoring, true. First of all, it shouldn't be organized as a class, and should look more like os-dependent CPython code. Not willing to bother with this right now, since it is not important at all.

Reply:
And, most importantly, it is out of scope for this PR :slightly_smiling_face: